### PR TITLE
Minor fixes to README_FUSE_INTEGRATION.md

### PR DESCRIPTION
**Pull request by AntStephenson**
Fixes to release/karaf/distro/README_FUSE_INTEGRATION.md

### CODE REVIEW

**Comment by ryanzhang** [comment]
Thanks Ant for this PR. This fix on README is request from QE.
Quoto Jiri's comments on the intpack release:
"3. Installation step 1: It might be useful to add the following line:
“features:addurl mvn:org.jboss.integration.fuse/karaf-features/${FUSE_INTEGRATION_VERSION}/xml/features”.
This line will add the list of int. pack features to Fuse. I know, that this file is imported by “mvn:org.jboss.integration.fuse.quickstarts/karaf-features/${FUSE_INTEGRATION_VERSION}/xml/features”. But I think customers may want to install integration pack on Fuse without adding features for quickstarts to their servers.
4. Installation step 2.: In the list of features is item called “fuse-bxms-jbpm-workitems-camel”. This feature does not exist."

and also update drools_version to avoid hardcode version in README.

File: release/karaf/distro/README_FUSE_INTEGRATION.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Camel components used in kie: kie-camel and jbpm-workitem-camel components
 
 Requirements
 ============
-DROOLS_VERSION --> 6.2.0.Final
+DROOLS_VERSION --> ${DROOLS_VERSION}
```
Comment (psiroky) [inline release/karaf/distro/README_FUSE_INTEGRATION.md]: This should be I believe removed or changed to something like `DROOLS_VERSION => 6.3.0.Final`. The current form does not add any additional info.

Comment (AntStephenson) [inline release/karaf/distro/README_FUSE_INTEGRATION.md]: Hi Petr,

We agree - I have amended this and pushed the change based on your feedback.

Best,

Ant

On 27/11/15 09:38, Petr Široký wrote:
>
> In release/karaf/distro/README_FUSE_INTEGRATION.md 
> <https://github.com/jboss-integration/fuse-bxms-integ/pull/172#discussion_r46030489>:
>
> > @@ -11,7 +11,7 @@ Camel components used in kie: kie-camel and jbpm-workitem-camel components
> >
> >  Requirements
> >  ============
> > -DROOLS_VERSION --> 6.2.0.Final
> > +DROOLS_VERSION --> ${DROOLS_VERSION}
>
> This should be I believe removed or changed to something like 
> |DROOLS_VERSION => 6.3.0.Final|. The current form does not add any 
> additional info.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/jboss-integration/fuse-bxms-integ/pull/172/files#r46030489>.
>

-- 
Ant Stephenson
Red Hat UK

**Comment by ryanzhang** [comment]
@psiroky  Ant have update the version to 6.3.x which should cover the versions on 6.3 product lines and also community release.

**Comment by dvirgiln** [comment]
I ve done a commit that filters the version of drools. You can check this PR:

https://github.com/jboss-integration/fuse-bxms-integ/pull/178

**Comment by ryanzhang** [comment]
Great! Thanks David for fixing it!

**Pull request closed without merging**